app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -135,9 +135,13 @@ def analyze
 
   def collect_compliance_data(image)
     return unless image_inspector_client.respond_to? :fetch_oscap_arf
+    _log.info "collecting compliance data for #{options[:docker_image_id]}"
     openscap_result = image.openscap_result || OpenscapResult.new(:container_image => image)
     openscap_result.attach_raw_result(image_inspector_client.fetch_oscap_arf)
     openscap_result.save
+  rescue ImageInspectorClient => e
+    _log.error("An error occurred while collecting compliance data name: #{options[:docker_image_id]} " \
```

Comment:
no need to split the line (even if Rubocop complains)
For symmetry, I would word it the same way as the first log line you added.